about.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>RESTAURANT-MENU</title>
+  <link rel="stylesheet" href="style.css">
+  <link=
+    href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Lato:ital,wght@0,300;0,700;1,700&display=swap"
+    rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+</head>
+
+<body>
+  <nav>
+    <div class="background">
+
+
+      <div class="first-bar">
+        <ul class="first-list">
+          <li><img src="images/facebook.svg" alt=""></li>
+          <li><img src="images/twitter.svg" alt=""></li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logout</li>
+        </ul>
+      </div>
+
+      <div class="bottom-nav">
+        <div class="top-nav">
+          <div id="harmburger">
+            <img src="images/menu.svg" alt="menu">
+          </div>
+        </div>
+      </div>
+
+      <div class="nav-bottom" id="menu">
+        <a href="index.html" class="nav-logo">
+          <span class="my-logo">
+            <img src="/images/main-logo.jpg" alt="mylogo">
+          </span>
+        </a>
+        <div class="close" id="cancel-btn">x</div>
+        <ul class="nav-menu">
+          <li> <a href="index.html">Home</a></li>
+          <li> <a href="#">Program</a></li>
+          <li> <a href="#">Join</a></li>
+          <li> <a href="#">News</a></li>
+          <li> <a href="#">Sponsor</a></li>
+          <li> <a class="campaign-item" href="#" id="button">CC Campaign</a></li>
+        </ul>
+      </div>
+
+
+    </div>
+    <!-- <div class="logo"><img src="./images/Restaurant-logo.jpg" alt=""></div> -->
```

Comment:
[REQUIRED] Kindly remove the commented-out code to make your project look more professional